server/controllers/verify.js

```diff
@@ -0,0 +1,23 @@
+const jwt = require('jsonwebtoken');
+require('env2')('./config.env');
+console.log('fffffffffffffffffffffffff');
+
+const verifyCookie = (req, res, next) => {
+  if (req.cookies.accessToken === undefined) {
+    console.log('notttttt allooe');
+    
+    // res.redirect('/');
+  } else {
+    jwt.verify(req.cookies.accessToken, process.env.JWT_KEY, (err, decoded) => {
```

Comment:
I think in login_post.js, you called the cookie `AdminToken` but here you're looking for a cookie called `accessToken`, so I think you'll need to make them match for this to work.